drivers/interrupt_controller/intc_nrf_egu.c

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sys/atomic.h>
+#include <kernel.h>
+#include <nrfx_egu.h>
+#include <drivers/interrupt_controller/intc_nrf_egu.h>
+
+#define DT_DRV_COMPAT nordic_nrf_egu
```

Comment:
Since DT instances cannot be used in this file, it is undesirable to set this macro.

Reply:
Removed the macro.